README.md

```diff
@@ -959,26 +959,6 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
```

Comment:
@calda if we remove this rule we have no guidance on `private` constants. Could we explore a replacement to cover this case so that our guide covers constants with any access control?

Reply:
Good point. I think in this case we should update the [namespace-using-enums](https://github.com/airbnb/swift#namespace-using-enums) rule to cover all cases, regardless of access control:

> Use caseless enums for organizing constants and functions into namespaces. Avoid creating non-namespaced global constants and functions. Feel free to nest namespaces where it adds clarity.

Reply:
We [revised](https://github.com/airbnb/swift/pull/106#discussion_r495278437) our thinking.